front/app/components/admin/PostManager/CommonGroundInputManager/index.tsx

```diff
@@ -0,0 +1,277 @@
+            disabled={!!isJobInProgress}
+          >
+            <FormattedMessage {...messages.startFromPastInputs} />
+          </Button>
+        </Box>
+      </NoPostPage>
+    );
+  }
+
+  const currentPage = getPageNumberFromUrl(ideas.links.self) || 1;
+  const lastPage = getPageNumberFromUrl(ideas.links.last) || 1;
+
+  return (
+    <>
+      {isJobInProgress && (
+        <Warning>
+          <Box display="flex">
+            <FormattedMessage
+              {...messages.inputImportProgress}
+              values={{ importedCount, totalCount: totalInputsToBeImported }}
+            />
+          </Box>
+        </Warning>
+      )}
+      <TopActionBar>
+        <StyledExportMenu
+          type={'ProjectProposals'}
+          selection={selection}
+          selectedProject={projectId}
+        />
+      </TopActionBar>
+      {selection.size > 0 && (
+        <Box mb="16px">
+          <ActionBar
+            selection={selection}
+            resetSelection={resetSelection}
+            handleClickEdit={openPreviewEdit}
+          />
+        </Box>
+      )}
+      <Table
+        border={`1px solid ${colors.grey300}`}
+        borderRadius={stylingConsts.borderRadius}
+        innerBorders={{ headerCells: true, bodyRows: true }}
+      >
+        <Thead>
+          <Tr background={colors.grey50}>
+            <Th>
+              <Checkbox
+                size="21px"
+                checked={allSelected}
+                onChange={toggleSelectAll}
+              />
+            </Th>
+            <Th>
+              <FormattedMessage {...messages.title} />
+            </Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {ideas.data.map((idea: IIdeaData) => (
+            <Tr
+              key={idea.id}
+              background={selection.has(idea.id) ? colors.grey300 : undefined}
+            >
+              <Td>
+                <Checkbox
+                  size="21px"
+                  checked={selection.has(idea.id)}
+                  onChange={() => toggleSelect(idea.id)}
+                />
+              </Td>
+              <Td>
+                <TitleLink
+                  className="e2e-common-ground-input-manager-title"
+                  onClick={(e) => {
+                    e.preventDefault();
+                    openPreview(idea.id);
+                  }}
+                >
+                  <T value={idea.attributes.title_multiloc} />
+                </TitleLink>
+              </Td>
+            </Tr>
+          ))}
+        </Tbody>
+      </Table>
+      {lastPage > 1 && (
+        <Box mt="16px">
+          <Pagination
+            currentPage={currentPage}
+            totalPages={lastPage}
+            loadPage={(page: number) =>
+              setQueryParameters({ ...queryParameters, 'page[number]': page })
+            }
+          />
+        </Box>
+      )}
+      <PostPreview
+        type={'ProjectIdeas' as ManagerType}
```

Comment:
I think "type" on the PostPreview is already ManagerType right? So we can remove this casting?